components/Typography.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
I think this file may have been deleted in the `ui-kitten` branch in favor of `components/Type`; we can cut it here if that's the case.